scripts/google_entities.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+# Update current Google entity IDs using ProPublica Congress API.
+
+import json, urllib.request, urllib.parse, urllib.error
+import rtyaml
+from utils import load_data, save_data
+
+def run():
+    # load in current members
+    y = load_data("/Users/DW-Admin/code/congress-legislators/legislators-current.yaml")
+    for m in y:
+        response = urllib.request.urlopen("http://propublica-congress.elasticbeanstalk.com/represent/api/v1/members/%s.json" % m['id']['bioguide']).read()
```

Comment:
Is this endpoint part of the official API? This domain isn't the one officially, documented, which is:  https://api.propublica.org/congress/

That host also has the benefit of HTTPS, which should be a near-requirement for our scripts whenever possible, given the value of data integrity for this project's role in providing upstream source data.